### DISCUSSION TITLE
Is jukebox mode / casting supposed to work with internet radios?

### DISCUSSION THREAD

**Discussion by gapato** [navidrome/navidrome#2654] [Q&A]
Trying to play internet radios (on DSub) and trying to "cast" to the server fails, although the radio plays as expected when not casting. Casting regular songs (not radios) works. Starting `mpv` from the command line and passing the radio's URL also works. Here are some errors from the log:

    Nov 24 21:53:42 XXX navidrome[588]: time="2023-11-24T21:53:42+01:00" level=warning msg="API: Failed response" endpoint=/rest/jukeboxControl.view error=70 message="data not found" requestId=XXX/wdY9XGeo6w-000102 username=YYY
    ...
    Nov 24 21:53:42 XXX navidrome[588]: time="2023-11-24T21:53:42+01:00" level=error msg="internal error: current song index out of bounds" idx=0 length=0
    Nov 24 21:53:42 XXX navidrome[588]: time="2023-11-24T21:53:42+01:00" level=warning msg="API: Failed response" endpoint=/rest/jukeboxControl.view error=0 message="Internal Server Error: could not get current track" requestId=XXX/wdY9XGeo6w-000103 username=YYY

Any ideas?

**Comment by metalheim** [comment]
I dont think so. In the API documentation, it specifially states that the Jukebox endpoint expect a `song` id

http://www.subsonic.org/pages/api.jsp#jukeboxControl

An `internetRadioStation` is a complettly different object than a `song` in Subsonic speak
